.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,29 +14,37 @@ Changelog:
 ## Types of changes
 
 What types of changes does your code introduce to Zowe Explorer?
-_Put an `x` in the boxes that apply_
+_Put an `x` in all boxes that apply._
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Enhancement (non-breaking change which adds or improves functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Updates to Documentation or Tests (if none of the other choices apply)
+- [ ] Documentation (Markdown, README updates)
+- [ ] Test cases (unit, integration, and/or end-to-end tests)
+- [ ] Other (please specify above in "Proposed changes" section)
 
 ## Checklist
 
-_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_
+_Put an `x` in all boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer._
+
+**General**
 
 - [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
-- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
-- [ ] PR Description is included
-- [ ] gif or screenshot is included if visual changes are made
+- [ ] All PR dependencies have been merged and published (if applicable)
+- [ ] A GIF or screenshot is included in the PR for visual changes
 - [ ] `pnpm --filter vscode-extension-for-zowe vscode:prepublish` has been executed
```

Comment:
I think the one shown on PRs comes from the main branch, should we include both with Zowe version in the main branch PR?